caluma/caluma_user/views.py

```diff
@@ -108,7 +113,7 @@ def get_user(self, request):
                     if "client_id" not in introspection:
                         response = HttpResponse(status=401)
                         raise HttpError(response)
-                    return models.OIDCUser(token=token, introspection=introspection)
+                    return self._oidc_user(token=token, introspection=introspection)
```

Comment:
Should be add some documentation about the two different cases? (userinfo, introspection)